install_gh.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+#################################
+# Installs github cli tool
+# Arguments:
+# None
+# Author: github.com/sosaeio
+# Date: Oct 28, 2023
+#################################
+
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+NC='\033[0m' # No Color
+CM=$(perl -e 'print "\xE2\x9C\x94"') #checkmark
+CURRDIR=$PWD
+
+# OS Specific Information
+OSTYPE=$(uname -s)
```

Comment:
Why are you setting variables? What if you just do everything functional?

Reply:
Removing this, found a better way to handle the path